src/Bicep.Core/TypeSystem/DeclaredTypeManager.cs

```diff
@@ -111,7 +111,7 @@ public DeclaredTypeManager(IResourceTypeProvider resourceTypeProvider, TypeManag
 
         private DeclaredTypeAssignment GetResourceType(ResourceDeclarationSyntax syntax)
         {
-            var declaredResourceType = syntax.GetDeclaredType(this.resourceTypeProvider);
+            var declaredResourceType = syntax.GetDeclaredType(this.binder, this.resourceTypeProvider);
```

Comment:
>GetDeclaredType [](start = 46, length = 15)

Are completions working ok inside the nested child resources? I'd expect them to but want to double check.

Reply:
Will be testing this out today.